Scripts/run-oclint.sh

```diff
@@ -61,18 +63,22 @@ if [ $TRAVIS ]; then
     full_sha=`git rev-parse $sha`
     echo $full_sha
     if [ ! -z "$include_files" ]; then
-      include_files="-i "$include_files      
+      include_files=' -i "'$include_files      
     else
       exclude_files="-e *"
     fi
-    echo "[*] $include_files"
-elif [ $1 ]; then
+    echo "[OCLint] analyzing these files: $include_files"
+elif [ $1 == "DIFF" ]; then
+    include_files=`git diff HEAD^ --name-only | grep '\.m' | tr '\n' '|' | sed 's/|*$/"/g'`    
```

Comment:
Check the following lines where I add a -I ".

On Friday, 30 January 2015, Sendhil Panchadsaram <notifications@github.com>
wrote:

> In Scripts/run-oclint.sh
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3153#discussion_r23818845>
> :
>
> >      else
> >        exclude_files="-e *"
> >      fi
> > -    echo "[*] $include_files"
> > -elif [ $1 ]; then
> > +    echo "[OCLint] analyzing these files: $include_files"
> > +elif [ $1 == "DIFF" ]; then
> > +    include_files=`git diff HEAD^ --name-only | grep '\.m' | tr '\n' '|' | sed 's/|*$/"/g'`
>
> Same comment as above
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wordpress-mobile/WordPress-iOS/pull/3153/files#r23818845>
> .
>